nemo/tron/tokenizers/tokenizer.py

```diff
@@ -8,15 +8,39 @@
 from pathlib import Path
 from typing import Dict, List, Optional
 
-from megatron.core.datasets.megatron_tokenizer import MegatronTokenizer
-from megatron.training.tokenizer.multimodal_tokenizer import MultimodalTokenizer
+from megatron.core.datasets.megatron_tokenizer import MegatronTokenizer as MegatronTokenizerCore
 
 from nemo.tron.config import TokenizerConfig
 from nemo.tron.tokenizers.bert_tokenization import FullTokenizer as FullBertTokenizer
 from nemo.tron.tokenizers.gpt2_tokenization import GPT2Tokenizer
+from nemo.tron.tokenizers.multimodal_tokenizer import MultimodalTokenizer
 from nemo.tron.utils.common_utils import get_rank_safe, print_rank_0
 
 
+class MegatronTokenizer(MegatronTokenizerCore):
```

Comment:
n00b question: why do we need this? in order to pass a tokenizer as a callable?

Reply:
Yes also to pass as a callable.